server/web/models.py

```diff
@@ -0,0 +1,30 @@
+from . import db
+from werkzeug.security import generate_password_hash, check_password_hash
+
+
+class Account(db.Model):
+    __tablename__ = 'account'
```

Comment:
Probably should be `accounts` table as each entry in the accounts table is an account